atrium/vestibulum/trcdb/trcplgtoolbase/trcplgtoolbase.go

```diff
@@ -124,6 +128,11 @@ func CommonMain(envPtr *string,
 		*pluginNameAliasPtr = driverConfig.DeploymentConfig["trcpluginalias"].(string)
 	}
 
+	if (len(*newrelicAppNamePtr) == 0 && len(*newrelicLicenseKeyPtr) != 0) || (len(*newrelicAppNamePtr) != 0 && len(*newrelicLicenseKeyPtr) == 0) {
```

Comment:
I think these need to be optional.

Reply:
This is to check whether both are empty or non-empty.